subprojects/kotlin-dsl-provider-plugins/src/main/kotlin/org/gradle/kotlin/dsl/provider/plugins/precompiled/tasks/GenerateScriptPluginAdapters.kt

```diff
@@ -83,6 +83,7 @@ fun PrecompiledScriptPlugin.writeScriptPluginAdapterTo(outputDir: File) {
          *
          * @see $compiledScriptTypeName
          */
+        @org.gradle.api.internal.Generated
```

Comment:
:o:  It would be better if the annotation would be put on the whole file instead `@file:Generated`. Would JaCoCo understand that? An integration test would prove that.

Reply:
For Accessors I had the same thought, but I didn't find yet where should I write that line, I was looking where the package is written without success.

Reply:
As discussed on Slack, `SourceFileHeader.kt` would be the place.